### ISSUE TITLE
EXC_BAD_ACCESS for NSManagedObjectContext in FBAllocationTrackerGeneration.mm

### ISSUE THREAD

**Issue by burniejm** [facebook/FBMemoryProfiler#26]
Crashes when checking for retain cycles in generation.  In my app I open a tableviewcontroller and then close it.  The table is backed by an NSFetchedResultsController.

Value of aCls is NSManagedObjectContext

![screenshot 2016-06-22 12 30 51](https://cloud.githubusercontent.com/assets/6455147/16276723/2b33258c-3875-11e6-9696-41b150a8f096.png)

not sure what other information will help :(

**Comment by Gricha** [comment]
Interesting, is this happening consistently? I am going to investigate.

**Comment by burniejm** [comment]
Yes I can reproduce this every time.  Are there any compiler settings to
watch out for?

On Wed, Jun 22, 2016 at 1:14 PM, Grzegorz Pstrucha <notifications@github.com
